charts/landscaper/charts/landscaper/values.yaml

```diff
@@ -8,6 +8,22 @@
 
 landscaper:
   verbosity: 2
+
+  controllers:
+    # syncPeriod: 10h
+    installations:
+      workers: 5
```

Comment:
Any reason this is 5 instead of the default 1?

Reply:
yes. for local development a default of 1 is better for debugging.
But in other cases when the controller is deployed via the helm chart I wanted to have sane defaults for productive or semi productive use cases.